index.html

```diff
@@ -0,0 +1,228 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```

Comment:
Please remove this line since you are not using in the project

Reply:
- [ ] Great work using inline comments. For this comment to be professional, kindly consider making the purpose of suggesting this change clear and understandable. 👍 💻 Please take a look at the following example:
Example of vague comment:
> The README is not descriptive, why don't you make it better?

Example of clear comment:
> The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?